Looks like VirtualBox 6.1.10 build is broken on current `nixos-unstable` (61525137fd10) when using `kernelPackages = pkgs.linuxPackages_latest`:
```
In file included from /nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/include/asm-generic/percpu.h:7,
                 from /nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/arch/x86/include/asm/percpu.h:556,
                 from /nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/arch/x86/include/asm/preempt.h:6,
                 from /nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/include/linux/preempt.h:78,
                 from /nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/include/linux/spinlock.h:51,
                 from /build/virtualbox-6.1.10-modsrc/vboxdrv/linux/../SUPDrvInternal.h:79,
                 from /build/virtualbox-6.1.10-modsrc/vboxdrv/linux/SUPDrv-linux.c:32:
/build/virtualbox-6.1.10-modsrc/vboxdrv/linux/SUPDrv-linux.c: In function 'supdrvOSChangeCR4':
/build/virtualbox-6.1.10-modsrc/vboxdrv/linux/SUPDrv-linux.c:760:38: error: 'cpu_tlbstate' undeclared (first use in this function); did you mean 'cpuhp_state'?
  760 |     RTCCUINTREG uOld = this_cpu_read(cpu_tlbstate.cr4);
      |                                      ^~~~~~~~~~~~
/nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/include/linux/percpu-defs.h:318:9: note: in definition of macro '__pcpu_size_call_return'
  318 |  typeof(variable) pscr_ret__;     \
      |         ^~~~~~~~
/build/virtualbox-6.1.10-modsrc/vboxdrv/linux/SUPDrv-linux.c:760:24: note: in expansion of macro 'this_cpu_read'
  760 |     RTCCUINTREG uOld = this_cpu_read(cpu_tlbstate.cr4);
      |                        ^~~~~~~~~~~~~
/build/virtualbox-6.1.10-modsrc/vboxdrv/linux/SUPDrv-linux.c:760:38: note: each undeclared identifier is reported only once for each function it appears in
  760 |     RTCCUINTREG uOld = this_cpu_read(cpu_tlbstate.cr4);
      |                                      ^~~~~~~~~~~~
/nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/include/linux/percpu-defs.h:318:9: note: in definition of macro '__pcpu_size_call_return'
  318 |  typeof(variable) pscr_ret__;     \
      |         ^~~~~~~~
/build/virtualbox-6.1.10-modsrc/vboxdrv/linux/SUPDrv-linux.c:760:24: note: in expansion of macro 'this_cpu_read'
  760 |     RTCCUINTREG uOld = this_cpu_read(cpu_tlbstate.cr4);
      |                        ^~~~~~~~~~~~~
make[4]: *** [/nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/scripts/Makefile.build:280: /build/virtualbox-6.1.10-modsrc/vboxdrv/linux/SUPDrv-linux.o] Error 1
make[3]: *** [/nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/Makefile:1756: /build/virtualbox-6.1.10-modsrc/vboxdrv] Error 2
make[2]: *** [/nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/source/Makefile:185: __sub-make] Error 2
make[2]: Leaving directory '/nix/store/lm27zw14mmif9prrslgawdi0i5wpx25v-linux-5.8.6-dev/lib/modules/5.8.6/build'
make[1]: *** [/build/virtualbox-6.1.10-modsrc/vboxdrv/Makefile-footer.gmk:114: vboxdrv] Error 2
make[1]: Leaving directory '/build/virtualbox-6.1.10-modsrc/vboxdrv'
make: *** [Makefile:58: vboxdrv] Error 2
builder for '/nix/store/1009iybgsnfr9p5vvkc19aajcxcqnklj-virtualbox-modules-6.1.10-5.8.6.drv' failed with exit code 2
cannot build derivation '/nix/store/cv7lscscdxnbnixn1dh7272az51qg7mf-kernel-modules.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/imcfygi6v5yvbd1pzav77nrj50k0z3w0-linux-5.8.6-modules.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/rjxwdazq3zff1iwb5pssgy5iql31wgkx-linux-5.8.6-modules-shrunk.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/yqq8z4y33g46yggpbah68crqd7xzj465-stage-1-init.sh.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/fh0jm66daaf4qnxpd1yx1qci05v36glm-initrd-linux-5.8.6.drv': 1 dependencies couldn't be built
```
The same configuration builds and runs fine with 6.1.14, so this is a very welcome change :slightly_smiling_face:

--- a/pkgs/applications/virtualization/virtualbox/default.nix
+++ b/pkgs/applications/virtualization/virtualbox/default.nix
@@ -23,7 +23,7 @@ let
   buildType = "release";
   # Use maintainers/scripts/update.nix to update the version and all related hashes or
   # change the hashes in extpack.nix and guest-additions/default.nix as well manually.
-  version = "6.1.10";
+  version = "6.1.14";
 
   iasl' = iasl.overrideAttrs (old: rec {
     inherit (old) pname;
@@ -40,7 +40,7 @@ in stdenv.mkDerivation {
 
   src = fetchurl {
     url = "https://download.virtualbox.org/virtualbox/${version}/VirtualBox-${version}.tar.bz2";
-    sha256 = "37d8b30c0be82a50c858f3fc70cde967882239b6212bb32e138d3615b423c477";
+    sha256 = "91fa05bcfce36316ca93e3927c9550ea66286fff4c5bec900b753fca278ce1a0";
   };
 
   outputs = [ "out" "modsrc" ];
@@ -120,6 +120,8 @@ in stdenv.mkDerivation {
     NIX_CFLAGS_COMPILE=$(echo "$NIX_CFLAGS_COMPILE" | sed 's,\-isystem ${lib.getDev stdenv.cc.libc}/include,,g')
 
     cat >> LocalConfig.kmk <<LOCAL_CONFIG
+    VBOX_WITH_VIRTIO               :=
+
     VBOX_WITH_TESTCASES            :=
     VBOX_WITH_TESTSUITE            :=
     VBOX_WITH_VALIDATIONKIT        :=

--- a/pkgs/applications/virtualization/virtualbox/extpack.nix
+++ b/pkgs/applications/virtualization/virtualbox/extpack.nix
@@ -12,7 +12,7 @@ fetchurl rec {
     # Manually sha256sum the extensionPack file, must be hex!
     # Thus do not use `nix-prefetch-url` but instead plain old `sha256sum`.
     # Checksums can also be found at https://www.virtualbox.org/download/hashes/${version}/SHA256SUMS
-    let value = "03067f27f4da07c5d0fdafc56d27e3ea23a60682b333b2a1010fb74ef9a40c28";
+    let value = "b224e796e886b19bce69f0aaedf6ca82bad0ca29c61fb0ed86166efb84356942";
     in assert (builtins.stringLength value) == 64; value;
 
   meta = {

--- a/pkgs/applications/virtualization/virtualbox/extra_symbols.patch
+++ b/pkgs/applications/virtualization/virtualbox/extra_symbols.patch
@@ -1,21 +1,25 @@
 diff --git a/src/VBox/HostDrivers/linux/Makefile b/src/VBox/HostDrivers/linux/Makefile
-index 6e44129b..e68ce128 100644
+index 8ba33952..3b8abe60 100644
 --- a/src/VBox/HostDrivers/linux/Makefile
 +++ b/src/VBox/HostDrivers/linux/Makefile
-@@ -95,13 +95,13 @@ vboxpci: vboxdrv
- install:
- 	@$(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) -C vboxdrv install
- 	@if [ -d vboxnetflt ]; then \
+@@ -99,17 +99,17 @@ install-vboxdrv:
+ 
+ install-vboxnetflt:
+ 	+@if [ -d vboxnetflt ]; then \
 -	    $(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) -C vboxnetflt install; \
-+		$(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) KBUILD_EXTRA_SYMBOLS=$(abspath vboxnetflt/Module.symvers) -C vboxnetflt install; \
++	    $(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) KBUILD_EXTRA_SYMBOLS=$(abspath vboxnetflt/Module.symvers) -C vboxnetflt install; \
  	fi
- 	@if [ -d vboxnetadp ]; then \
+ 
+ install-vboxnetadp:
+ 	+@if [ -d vboxnetadp ]; then \
 -	    $(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) -C vboxnetadp install; \
 +	    $(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) KBUILD_EXTRA_SYMBOLS=$(abspath vboxnetadp/Module.symvers) -C vboxnetadp install; \
  	fi
- 	@if [ -d vboxpci ]; then \
+ 
+ install-vboxpci:
+ 	+@if [ -d vboxpci ]; then \
 -	    $(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) -C vboxpci install; \
 +	    $(MAKE) KBUILD_VERBOSE=$(KBUILD_VERBOSE) KBUILD_EXTRA_SYMBOLS=$(abspath vboxpci/Module.symvers) -C vboxpci install; \
  	fi
  
- else
+ install: install-vboxdrv install-vboxnetflt install-vboxnetadp install-vboxpci

--- a/pkgs/applications/virtualization/virtualbox/guest-additions/default.nix
+++ b/pkgs/applications/virtualization/virtualbox/guest-additions/default.nix
@@ -27,7 +27,7 @@ in stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "http://download.virtualbox.org/virtualbox/${version}/VBoxGuestAdditions_${version}.iso";
-    sha256 = "62a0c6715bee164817a6f58858dec1d60f01fd0ae00a377a75bbf885ddbd0a61";
+    sha256 = "dd9f176abb89043c01cea7ec7e20130e76db71bd83beafeb2dc5858d4c9c86cd";
   };
 
   KERN_DIR = "${kernel.dev}/lib/modules/${kernel.modDirVersion}/build";

